analyses/mutect2-vs-strelka2/01-set-up.Rmd

```diff
@@ -0,0 +1,256 @@
+---
+title: "Evaluate concordance between Mutect2 and Strelka2"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+Candace Savonen - CCDL for ALSF
+
+This notebook is a first pass in evaluating the concordance between MuTect2 and 
+Strelka2 results.
+
+It addresses [issue \# 30 in OpenPBTA](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/30).
+
+## Set Up
+
+\* Some of these set up steps will be removed once we have a Dockerfile that installs
+maftools automatically. 
+
+```{r}
+# We need maftools - this will be added to the running Docker issue whenever it is up
+if (!('maftools' %in% installed.packages())) {
+  devtools::install_github("PoisonAlien/maftools")
```

Comment:
@cansav09 I am working on #33 and was planning on using `maftools` to do so. Is there a reason you're installing what I am assuming is a development version of the package from GitHub here rather than installing from bioconductor? If you're using functionality that is not available from the bioconductor version, you should tie this to a specific commit via the `ref` argument to `devtools::install_github`.

Reply:
I switched versions of `maftools` when I was having problems running `maftools` in Docker and hadn't yet identified that it was definitely an OOM problem. I will return to `maftools_2.0.15` which is the version that will be default installed with our tidyverse 3.6.0 Docker image and make sure everything run smoothly with this version. When this PR is ready for prime time, I'll make sure `maftools` is added installation to the Dockerfile.